packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/meshes/meshPropertyGridComponent.tsx

```diff
@@ -474,6 +475,21 @@ export class MeshPropertyGridComponent extends React.Component<
                             onPropertyChangedObservable={this.props.onPropertyChangedObservable}
                         />
                     )}
+                    <OptionsLineComponent
+                        label="Orientation"
+                        target={mesh}
+                        propertyName="sideOrientation"
+                        options={[
+                            { label: "None", value: "" },

```

Comment:
I'm guessing you are using `""` because using `null` doesn't work? This seems complicated to handle with all the branching in the related code.

Reply:
Can we update it to work? It seems like an issue with the component.